var/spack/repos/builtin/packages/dyninst/package.py

```diff
@@ -67,7 +67,11 @@ class Dyninst(CMakePackage):
     # libdwarf before that.
     depends_on("libdwarf", when="@:9")
 
-    depends_on("tbb@2018.6:", when="@10.0.0:")
+    # findtbb.cmake in the dynist repo does not work with recent tbb
+    # package layout. Need to use tbb provided config instead.
+    conflicts("intel-tbb@2021.1:")
+    conflicts("intel-oneapi-tbb@2021.1:")
+    depends_on("tbb@2018.6.0:", when="@10.0.0:")
```

Comment:
It was like that before my change so I left it.

I guess this would work if intel-tbb and intel-oneapi-tbb did:
provides('tbb@2018.6.0', when='@2018.6.0')
but all I am doing is
provides('tbb')